<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Raschet</class>
 <widget class="QWidget" name="Raschet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Метод гаммирования</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Заполните значения для ПСЧ:</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_A">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="text">
    <string>A = </string>
   </property>
  </widget>
  <widget class="QLabel" name="text_C">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>C =</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_C">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_A">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="text_m">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Начальное знач. ПСЧ:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_m">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="text_vvod">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите исходный текст:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="line_vvod">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_shifr">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>зашифравать</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_shifr">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>зашифрованный текст:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="line_shifr">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_rashifr">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Расшифровать</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="line_enter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="text_enter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Расшифрованный текст:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>Raschet</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
